iina/PlayerCore.swift

```diff
@@ -667,30 +667,40 @@ class PlayerCore: NSObject {
     syncUITime()
   }
 
-  func setCrop(fromString str: String) {
+  func setCrop(fromString str: String) -> Bool? {
```

Comment:
I don't see you trying to handle `nil` anywhere, so why not make the function `throw` an error instead?